### PR TITLE
[ADF-4582] User preference - set layout orientation based on locale

### PR DESCRIPTION
**Pull request by pionnegru**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
[ADF-4582](https://issues.alfresco.com/jira/browse/ADF-4582)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/services/user-preferences.service.ts [hunk 1]
```diff
@@ -212,7 +218,14 @@ export class UserPreferencesService {
      * @returns Default locale language code
      */
     public getDefaultLocale(): string {
-        return this.appConfig.get<string>(UserPreferenceValues.Locale) || this.translate.getBrowserCultureLang() || 'en';
+        return this.appConfig.get<string>(UserPreferenceValues.Locale) || this.translate.getBrowserLang() || 'en';
```
Comment (DenysVuika) [inline lib/core/services/user-preferences.service.ts]: is that going to work with simplified Chinese and pt-BZ ? I remember we added the "getBrowserCultureLang" call to fix a few issues...

Comment (pionnegru) [reply lib/core/services/user-preferences.service.ts]: i'll revert

File: lib/core/services/user-preferences.service.ts [hunk 2]
```diff
@@ -215,4 +221,11 @@ export class UserPreferencesService {
         return this.appConfig.get<string>(UserPreferenceValues.Locale) || this.translate.getBrowserCultureLang() || 'en';
     }
 
+    private getLanguageByKey(key: string) {
```
Comment (DenysVuika) [inline lib/core/services/user-preferences.service.ts]: missing return type

File: lib/core/services/user-preferences.service.ts [hunk 3]
```diff
@@ -215,4 +221,11 @@ export class UserPreferencesService {
...
+        return (
+            this.appConfig
+                .get<Array<any>>(AppConfigValues.APP_CONFIG_LANGUAGES_KEY)
```
Comment (DenysVuika) [inline lib/core/services/user-preferences.service.ts]: Missing exact type of the value

**Comment by pionnegru** [comment]
@DenysVuika updated

**Pull request merged**